src/contracts/index.ts

```diff
@@ -1,8 +1,14 @@
 /* Autogenerated file. Do not edit manually. */
 /* tslint:disable */
 /* eslint-disable */
-export type { Forwarder } from "./Forwarder";
-export type { GaslessWallet } from "./GaslessWallet";
+export type { AvoAuthoritiesList } from "./AvoAuthoritiesList";
+export type { AvoDepositManager } from "./AvoDepositManager";
+export type { AvoFactory } from "./AvoFactory";
```

Comment:
@bergben I think we don't have these

Reply:
oh sorry about this. Fixed it https://github.com/Instadapp/avocado-sdk/pull/6/commits/f91f96eefac2cf77f0572d10f729b0f7f8b5ea9b